addons/spreadsheet/static/tests/migrations/migrations.test.js

```diff
@@ -618,3 +618,19 @@ test("text global filter default value is now an array of strings", () => {
     expect(migratedData.globalFilters[1].rangeOfAllowedValues).toBe(undefined);
     expect(migratedData.globalFilters[1].rangesOfAllowedValues).toBe(undefined);
 });
+
+test("Date with antepenultimate_year is not supported anymore", () => {
```

Comment:
already tested in "Global filters: date is correctly migrated"

Reply:
Yes, but I want to have an explicit test (in case someone later ask himself why it's not supported anymore, he will find this commit